test/graphql.test.ts

```diff
@@ -196,6 +196,17 @@ describe('graphql', () => {
     expect(paginatedQuery.pageInfoPath).toBeEmpty();
   });
 
+  test('paginated where v2 query', async () => {
```

Comment:
let's also test what comes out of 
```typescript
const query =
      await loadQueryFile('paginated-commits-where-v2.gql');
const paginatedQuery = sut.paginateWithWhereV2(query);
// ... assertions   
```

Reply:
What do you mean?

Reply:
We might want to test what happens with existing arguments if they were provided in the query (see - https://github.com/faros-ai/faros-js-client/pull/142#discussion_r1196678700)